apps/prairielearn/src/components/PageLayout.html.tsx

```
@@ -0,0 +1,400 @@
+                resLocals={resLocals}
+                navPage={navContext.page}
+                navSubPage={navContext.subPage}
+                navbarType={navContext.type}
+                isInPageLayout
+              />
+            </div>
+            {showSideNav ? (
+              <div
+                class="app-side-nav"
+                // eslint-disable-next-line @eslint-react/dom/no-dangerously-set-innerhtml
+                dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)